src/ansiblelint/rules/var_naming.md

```diff
@@ -7,7 +7,9 @@ underscore `_` character. Variable names must also start with either an
 alphabetic or underscore `_` character.
 
 For more information see the [creating valid variable names][var-names] topic in
-Ansible documentation and [Naming things (Good Practices for Ansible)][cop].
+Ansible documentation and [Naming things (Good Practices for Ansible)][cop]. You
+should also be fully aware of [special variables][magic-vars], also known as
+magic variables, especially as most of them can only be read.
```

Comment:
maybe inform that 'while it is possible to assign to them w/o an error, this will be ignored'